### PR TITLE
Sync upstream release 1.5

### PR DESCRIPTION
**Pull request by hasheddan**
### Description of your changes

Syncs `upbound-crossplane@release-1.5` with `crossplane@release-1.5`.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

`make reviewable`

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Review by hasheddan — COMMENTED** [review]
Seeing issues with `Init:ErrImageNeverPull`, which I believe is due to Makefile changes associated with switch to `imagelight.mk`.

**Review by hasheddan — COMMENTED** [review]
Confirmed, we are loading image into `kind` cluster with `PROJECT_NAME`, which is `crossplane`, but using image with org `upbound`:

```
Image: "crossplane/crossplane:v1.5.1-up.1.44.g936121bc.dirty" with ID "sha256:a963a468743bd551cc27cc01ada01f751fda9738950351b6c55ca6822342e9b1" not yet present on node "build-ff484a15-inttests-control-plane", loading...
```

```
Containers:
  crossplane:
    Container ID:  
    Image:         upbound/crossplane:v1.5.1-up.1.44.g936121bc.dirty
```

I think we should use a temporary fix here to avoid having to modify `PROJECT_NAME`.

**Comment by hasheddan** [comment]
Also noting that @turkenh fixed this in `master` and `release-1.6` with https://github.com/upbound/crossplane/commit/5d53beb3cb423b13960a92d1f8f9284c9a146ccc

File: .github/workflows/configurations.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+name: Configurations
```
Comment (turkenh) [inline .github/workflows/configurations.yml:1]: Same as https://github.com/upbound/crossplane/pull/33#discussion_r805398991

**Review by turkenh — APPROVED** [review]
Thanks @hasheddan !

File: .github/workflows/configurations.yml (hunk 1 above)
Comment (hasheddan) [reply .github/workflows/configurations.yml]: Removed :)

**Pull request merged**